src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -1138,7 +1251,16 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
     {
       renderWindow->SetSkyEnabled(true);
       if (!elem->NoChildren())
-        ignwarn << "Child elements of <sky> are not supported yet" << std::endl;
+        ignwarn << "Child elements of <sky> are not supported yet"
+            << std::endl;
+    }
+
+    elem = _pluginElem->FirstChildElement("graphics_api");
```

Comment:
can you add a brief comment documenting this new param here?
https://github.com/ignitionrobotics/ign-gui/blob/5cef8a1b3f2550d3e380e0ed12d88e1a8c890430/src/plugins/minimal_scene/MinimalScene.hh#L61

Reply:
Updated in ca52977561c63bb615e01b25fa62b350645dfda6